inlineplz/main.py

```diff
@@ -27,6 +27,7 @@ def main():
     parser.add_argument('--zero-exit', action='store_true')
     parser.add_argument('--install', action='store_true')
     parser.add_argument('--max-comments', default=25, type=int, help='maximum comments to write')
+    parser.add_argument('--prefix', type=str)
```

Comment:
would it maybe be better to set the default prefix here? also maybe name this `--comment-prefix`? or is that too wordy?